dhall-json/src/Dhall/JSON/Yaml.hs

```diff
@@ -45,7 +46,7 @@ defaultOptions =
           , omission = id
           , documents = False
           , quoted = False
-          , conversion = NoConversion
+          , conversion = Dhall.JSON.defaultConversion
```

Comment:
I believe there are more aspects where these `defaultOptions` diverge from the default behaviour of `dhall-yaml[-ng]`, for example `omission`. I think it would be nice if these `defaultOptions` would exactly match the CLI defaults, and were in fact used by the executables.

I wonder whether we should also try to share these `Options` between `dhall-json` and `dhall-yaml[-ng]`…

Reply:
Yeah, I think this is fine.  My understanding is that `dhall-yaml` already has a dependency on `dhall-json` anyway

Reply:
Right. `dhall-to-json` currently has it's own `Options` record:

https://github.com/dhall-lang/dhall-haskell/blob/85645a2470820b51632504e0086bcfbfadedb717/dhall-json/dhall-to-json/Main.hs#L27-L37

The fields that it shares with `dhall-to-yaml[-ng]` could in principle live in a common `DhallToXOptions` record or something like that.